### ISSUE TITLE
simplify update_astrerisk_sounds and update_asterisk_sounds_virtualbox

### ISSUE THREAD

**Issue by kra** [futel/asteriskserver#436]
before having two different ways of deploying, we need to simplify the sound population:

use with_items
use fewer source directories
- can we just do src/var/lib/asterisk/sounds, src/var/lib/asterisk/moh, and create var/lib/asterisk/sounds/futel/recordings?

the end result should be to make the two ansible files as simliar as possible
we should be able to easily tell whether they are different

this will need to be tested with a stage deploy before continuing. The reason the subdirs were split out so much was because it sometimes stalls on a deploy to stage.

**Comment by kra** [comment]
ok, with_items and a list of subdirs will work if we can't just do the top level. Maybe put that list in conf.yml to share it between the two deploys.

**Comment by lboom** [comment]
@kra If we can't or don't want to deploy a stage from my branch I can make a PR for this. Check out my branch to see what I am thinking here. https://github.com/kra/futel-installation/compare/tishbite_simplify_sounds?expand=1

**Comment by lboom** [comment]
Since the MOH sounds are so small we probably don't need to move them to the content repo. I am copying them here for compatibility. I could reduce the items lists as well if you think thats a good idea... this was the result of me using copies and links depending on deployment so its not really needed.

**Comment by kra** [comment]
I don't understand the branch, I commented. But what I mean by this issue is to simplify the sound population recipes as much as possible to prepare for using the volume.

Right now the volume isn't populated yet and there are more tasks that have to be done before that happens, I want the recipe cleaned up if possible to make that process more foolproof. The recipes are kind of crap because I built them over time while learning ansible.

I'd like to have the volume completely populated and ready and deployed alongside the current copied files, while the current copied files are still in use. Then when we want to change over we can just have one commit that changes how asterisk gets the assets, like by replacing copy tasks with link tasks or whatever. Then if there is an issue I can just throw away the stage, back out the commit, redeploy stage, and not impact having a deployable stage, hopefully.

So for example if you just committed the parameterization change first.
